src/ert/ensemble_evaluator/_builder/_ensemble.py

```diff
@@ -111,6 +111,9 @@ def __repr__(self) -> str:
     def evaluate(self, config: "EvaluatorServerConfig") -> None:
         pass
 
+    async def evaluate_async(self, config: "EvaluatorServerConfig") -> None:
```

Comment:
Should `Ensemble` be an abstract base class instead?

Reply:
Maybe it should. I'd also rename the `_LegacyEnsemble` to `Ensemble` and `Ensemble` could be `BaseEnsemble` or something similar. But it should be another PR.